src/CharacterList.tsx

```diff
@@ -16,51 +14,31 @@ interface Vehicle {
 
 const ListItem = ({ person }: { person: Character }): JSX.Element => {
 
-  // const [vehicles, setVehicles] = useState<Vehicle[]>([])
-
-  // const getData = async (url: string) => {
-  //   const { data } = await axios.get(url);
-  //   return data;
-  // }
-
-  // useEffect(() => {
-  //   const vehicleData = person.vehicles.map(x => getData(x))
-  //   console.log(vehicleData)
-  //   Promise.all(vehicleData).then(x => setVehicles(x))
-  // }, [])
-
-  const query = (vehicleUrl: string) => {
-    return fetch(vehicleUrl).then(res => res.json())
-  }
-
-  const { isLoading, error, data } = useQuery(["getVehicle", vehicleUrl], () => query(vehicleUrl))
-
-  // console.log({ isLoading, data })
-
-  const queries = person.vehicles.map((v, i) => {
-    return {
-      queryKey: ["vehicle", i + 1],
-      queryFn: () => {
-        fetch(v).then(res => res.json())
+  const vehicleData = useQueries<Vehicle[]>(
+    person.vehicles.map((v, i) => {
+      return {
+        queryKey: ["vehicle", i + 1],
+        queryFn: () => {
+          return fetch(v).then(res => res.json())
+        }
       }
-    }
-  })
+    })
+  )
 
-  const { isLoading, error, data } = useQueries({
-    queries: queries
-  })
+  const loading = vehicleData.some(v => v.isLoading)
```

Comment:
as useQueries returns an array of the classic useQuery result i.e. `{data, isLoading, error} = useQuery(blah)` we want to map/filter them to get a value for isLoading, or a mapped array of just the vehicles. and you can type them as such